SU-SWS/stanford_bean_types
# HSDO-524 added a bean status capability

# READY FOR REVIEW

# Summary
- Added a publishing (status) option to beans

# Needed By (Date)
- 4/12

# Steps to Test

1. Checkout branch, clear cache
2. edit a bean. Near the bottom should be a publish fieldset similar to nodes
3. unpublish a bean and view the page it was on as anonymous.
4. some css should apply to indicate the block is unpublished to people with access to unpublish/publish it.

# See Also
[Field Formatters](https://github.com/SU-SWS/stanford_field_formatters/pull/2)
